### ISSUE TITLE
go learn something instead of pasting

### ISSUE THREAD

**Issue by SamHoque** [Activx-nV/portfolio-current#1]
Good job pasting my portfolio site, https://samhoque.dev, you didn't even have the knowledge to fix the cdn icons nor get swiper to work with it.

**Comment by Activx-nV** [comment]
Hey, had to quickly find a good-looking template. The only thing I can tell is that it's very great, the colors are well-chosen, someday I'll replace it dw ;)

**Comment by SamHoque** [comment]
It's not a template, It's my own work. You should make your own project since I did not give you consent to use mine.

**Comment by Activx-nV** [comment]
Right, sorry. I will remove it from here over the next few days, so you can see this message before deletion.
